virtcontainers/qemu_arch_base.go

```diff
@@ -475,6 +475,24 @@ func (q *qemuArchBase) appendNetwork(devices []govmmQemu.Device, endpoint Endpoi
 			},
 		)
 		q.networkIndex++
+	case *MacvtapEndpoint:
+		devices = append(devices,
+			govmmQemu.NetDevice{
+				Type:          govmmQemu.MACVTAP,
+				Driver:        govmmQemu.VirtioNetPCI,
+				ID:            fmt.Sprintf("network-%d", q.networkIndex),
+				IFName:        ep.Name(),
+				MACAddress:    ep.HardwareAddr(),
+				DownScript:    "no",
+				Script:        "no",
+				VHost:         true,
```

Comment:
Same comment as before, couldn't we use `fallthrough` here instead?
I can see the difference is `Type: govmmQemu.MACVTAP,` in this case, but we need a simple helper function to keep this generic. The same way virtual endpoint and macvlan uses `networkModelToQemuType()`.

Reply:
We agreed offline that this cannot be generalized further.